android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/Email.kt

```diff
@@ -89,46 +86,69 @@ class Email {
 
     fun scrape(call: PluginCall) {
```

Comment:
This method needs to loop through each account and call the .messages method. Check the example in https://github.com/BlinkReceipt/blinkreceipt-android/blob/485d4f3f66c6938c976e8f1ff7510f851e5742d8/blinkreceipt-demo/BlinkReceiptImap/src/main/java/com/blinkreceipt/digital/imap/MainActivity.kt#L129